components/CollaboratorInput/CollaboratorValues.tsx

```diff
@@ -1,4 +1,7 @@
 import { useContractBuilderProvider } from '@/providers/ContractBuilderProvider'
+import { CollaboratorTypeDropdown } from './CollaboratorTypeDropdown'
+
+const intoClass = 'font-share text-base/4 pb-1 text-[15px] text-[#696969]'
```

Comment:
Could you please add this color code to the `tailwind` config file and use `human-readable` color names???